Makefile

```diff
@@ -50,6 +50,7 @@ test-e2e-ci: build $(src) $(tests)
 	@$(wdio) wdio.ci.conf.js
 test-types: types
 	$(dts) test/types
+	$(dts) types
```

Comment:
This is the change that adds linting for the types themselves in CI with the same rules on DefinitelyTyped

Reply:
Let's add an @ before to silence the cmd

Reply:
Updated https://github.com/recurly/recurly-js/blob/014c2a65c83b30bd6d351972ab91a68dc16f6b0c/Makefile#L53